### ISSUE TITLE
Is requiring exact 2.7 python version really intended?

### ISSUE THREAD

**Issue by blattms** [Statoil/libecl#15]
CMake deactivates python on my system with
```
-- Found PythonInterp: /usr/bin/python (found version "2.7.9") 
-- Could NOT find PythonInterp: Found unsuitable version "2.7.9", but required is exact version "2.7" (found /usr/bin/python)
```
Is this intended?

**Comment by joakim-hove** [comment]
No - that is not intended; but are you sure the message is not a false positive. At least in Statoil we get exactly the same message falsely on Python 2.7.c

**Comment by markusdregi** [comment]
Fixed in #18.
